### PR TITLE
OrderUp case study; case study style changes; case study image reorg

### PR DESCRIPTION
**Pull request by yflicker**
OrderUp case study is here.

BUT I got a little carried away and also changed the styling of the case studies - I thought there was way too much whitespace, especially on table/mobile view—this did involve me eliminating the company logo from the desktop view (still there on table+phone).

I also changed how case study images are organized—I just put all the different case study image folders into another folder.

@ryandestefano @danivovich @BSierakowski or someone else please review and give it a thumbs up before I merge.

### CODE REVIEW

File: case-study/brookes-publishing.html [hunk 1]
```diff
@@ -30,9 +30,9 @@
...
-      <li>Using <a href="http://blog.smartlogicsolutions.com/2012/11/28/using-pull-requests-to-improve-code-quality-process-and-skills/">pull requests</a> with two approvals to improve code quality.</li>
-      <li>Chatting using Campfire to <a href="http://blog.smartlogicsolutions.com/2012/08/09/what-types-of-meetings-make-a-successful-application-development-project/">streamline communication</a>, which can allow us to have fewer formal meetings than in a typical development process.</li>
+      <li>Tracking development via <a href="http://blog.smartlogic.io/2013/04/12/how-user-stories-help-software-developers-know-what-the-___-to-do/">user stories</a> in Pivotal Tracker.</li>
+      <li>Using <a href="http://blog.smartlogic.io/2012/11/28/using-pull-requests-to-improve-code-quality-process-and-skills/">pull requests</a> with two approvals to improve code quality.</li>
+      <li>Chatting using Campfire to <a href="http://blog.smartlogic.io/2012/08/09/what-types-of-meetings-make-a-successful-application-development-project/">streamline communication</a>, which can allow us to have fewer formal meetings than in a typical development process.</li>
```
Comment (samwgoldman) [inline case-study/brookes-publishing.html]: I'm getting 404s on the above links (before and after this change).

File: case-study/dinnertime.html [hunk 2]
```diff
@@ -32,22 +32,22 @@
...
   </figure>
   <p>SmartLogic and DinnerTime have proved to be excellent partners. Here are a few of DinnerTime’s favorite aspects of working with SmartLogic:</p>
   <ul>
     <li>DinnerTime was impressed with the "rigorous disciplines SmartLogic utilizes to plan, coordinate and manage rapid iterations and feature builds," said Lay.  These processes include:
     <ul>
-      <li>Tracking development via <a href="http://blog.smartlogicsolutions.com/2013/04/12/how-user-stories-help-software-developers-know-what-the-___-to-do/">user stories</a> in Pivotal Tracker.</li>
-      <li>Using <a href="http://blog.smartlogicsolutions.com/2012/11/28/using-pull-requests-to-improve-code-quality-process-and-skills/">pull requests</a> with two approvals to improve code quality.</li>
+      <li>Tracking development via <a href="http://blog.smartlogic.io/2013/04/12/how-user-stories-help-software-developers-know-what-the-___-to-do/">user stories</a> in Pivotal Tracker.</li>
+      <li>Using <a href="http://blog.smartlogic.io/2012/11/28/using-pull-requests-to-improve-code-quality-process-and-skills/">pull requests</a> with two approvals to improve code quality.</li>
```
Comment (samwgoldman) [inline case-study/dinnertime.html]: Ditto 404

File: case-study/brookes-publishing.html [hunk 3]
```diff
@@ -12,14 +12,14 @@
...
-  <img src="{{ site.baseurl }}/images/brookes-case-study/brookes-02.png">
+  <p>A large part of why Gebhardt and her team felt comfortable with SmartLogic was that SmartLogic could translate between Brookes Publishing’s language and the technical language of software development. "They were easily able to translate our needs into development language, and they relayed technical information in a way that we could understand it and present it to our executives," said Gebhardt. <a href="http://blog.smartlogic.io/2012/08/09/what-types-of-meetings-make-a-successful-application-development-project/">Communication</a> has been one of the best parts of SmartLogic’s relationship with Brookes Publishing moving forward. Clear communications save Brookes Publishing stress, time, and money.</p>
```
Comment (samwgoldman) [inline case-study/brookes-publishing.html]: Blog link broken

File: case-study/dinnertime.html [hunk 4]
```diff
@@ -32,22 +32,22 @@
...
       <li>Employing continuous integration to rapidly deploy new code.</li>
     </ul>
     </li>
     <li>These processes, said Lay, give him "terrific visibility on key priorities." When necessary, Lay can "easily and quickly adjust development priorities" as the company pivots, which is key to any growing company’s success.</li>
     <li>Lay is "in near constant communication with the SmartLogic team," and DinnerTime’s client-facing management team is in touch frequently as well. Lay praises this "highly collaborative environment." SmartLogic emphasizes communication and collaboration via the following methods:
     <ul>
-      <li>Holding <a href="http://blog.smartlogicsolutions.com/2012/11/28/using-pull-requests-to-improve-code-quality-process-and-skills/">epic, iteration, and daily meetings or phone calls</a> to establish big priorities and shift them as needed on an ongoing basis.</li>
+      <li>Holding <a href="http://blog.smartlogic.io/2012/11/28/using-pull-requests-to-improve-code-quality-process-and-skills/">epic, iteration, and daily meetings or phone calls</a> to establish big priorities and shift them as needed on an ongoing basis.</li>
```
Comment (samwgoldman) [inline case-study/dinnertime.html]: Last one

**Comment by leannematlach** [comment]
I fixed all the links and verified they work. Can someone re-review this pull request.

**Comment by danivovich** [comment]
I guess these links got fixed on the Hubspot side? The few I tested worked for me now.

There are some issues with the company logo, but only in firefox
screen shot 2015-02-05 at 7 24 17 am

It isn't unique to the OrderUp case-study
screen shot 2015-02-05 at 7 24 50 am

It gets strange at smaller resolutions until it changes orientation
screen shot 2015-02-05 at 7 25 00 am

**Comment by danivovich** [comment]
Also, I don't think the OrderUp case study is actually linked to from the experience page

**Comment by yflicker** [comment]
Yeah the links were fixed on the hubspot side. And good catch, it is not actually linked to from the experience page.

**Pull request merged**